packages/core/minos-microservice-common/minos/common/object.py

```diff
@@ -0,0 +1,5 @@
+class Object:
```

Comment:
The reason to add this class is to avoid the Cooperative Multiple Inheritance issues as proposed at: https://stackoverflow.com/a/19012157/3921457